include/revng-c/mlir/Dialect/Clift/IR/CliftOps.td

```diff
@@ -74,7 +74,7 @@ def Clift_FunctionOp : Clift_Op<"func",
                                  IsolatedFromAbove,
                                  NoTerminator]> {
   let arguments = (ins StrAttr:$sym_name,
-                       TypeAttrOf<Clift_FunctionType>:$function_type,
+                       TypeAttrOf<Clift_AnyFunctionType>:$function_type,
```

Comment:
Can you remind me the rationale of this rename?

Reply:
It's just for consistency. I'm naming all type constraints following this pattern.